core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -39,6 +39,8 @@
 import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
+import com.google.common.collect.BiMap;
```

Comment:
We used to remove the usages of guava collect package, because GMS core doesn't like it, I'm not very sure about the current status, but I'd assume they still don't like it.